### PR TITLE
PLG-121: Track checkout event

### PR DESCRIPTION
**Pull request by nedick007**
Simple class responsible for tracking checkout start process.

### CODE REVIEW

**Comment by nedick007** [comment]
@adamof ready for review

**Comment by nedick007** [comment]
@adamof bump

File: etc/di.xml [hunk 1]
```diff
@@ -19,4 +19,9 @@
             <argument name="catalogSearchEvent" xsi:type="object">Metrilo\Analytics\Model\Events\CatalogSearch\Proxy</argument>
         </arguments>
     </type>
+    <type name="Metrilo\Analytics\Block\Analytics">
```
Comment (adamof) [inline etc/di.xml]: Why do you need this pattern for this class? It doesn't load anything in the constructor?

Comment (nedick007) [reply etc/di.xml]: No idea what i was thinking tbh... Will remove it.

**Comment by nedick007** [comment]
Tested on QA env.

**Pull request merged**